js/services.js

```diff
@@ -199,15 +199,14 @@ app.service('recoveryServices', ['$rootScope', '$http', 'lodash',
         var derivedPublicKey = derivedHdPublicKey.publicKey;
         derivedPublicKeys.push(derivedPublicKey);
       });
-
       var address;
+
       if (wallet.addressType == "P2SH")
-        address = bitcore.Address.createMultisig(derivedPublicKeys, wallet.n, wallet.network);
+        address = bitcore.Address.createMultisig(derivedPublicKeys, parseInt(wallet.m), wallet.network);
```

Comment:
`wallet.m` & `wallet.n` should already be numeric. Either parse it when pulling the data out of the form or at wallet initialization in `fromXXX()` methods.